### ISSUE TITLE
Hvordan skifter man den ansvarlige for processen?

### ISSUE THREAD

**Issue by DCRGraphs** [DCRGraphsNet/DCROpenCaseManager#327]
Altså den der er ansvarlig for at udfører aktiviteterne?

**Comment by NicklasHealySyddjurs** [comment]
@DCRGraphs ; Vil du sende dokumentation for hvordan den skal fungere?

**Comment by NicklasHealySyddjurs** [comment]
![image](https://user-images.githubusercontent.com/47558591/60428368-f8361780-9bf8-11e9-81b1-3dadb898f7a5.png)

**Comment by NicklasHealySyddjurs** [comment]
http://opencasemanagertest.syddjurs.dk/Child?id=679442

Skal fjernes hvis tomme.

**Comment by NicklasHealySyddjurs** [comment]
Test følgende stedet
- [x]Fra barnets side - under barnets navn
- [ ]Fra barnets side - i oversigten på en indsats
- [ ]Fra indsatssiden – under indsatstitlen
- [ ]Fra indsatssiden – på aktiviteten

Et flyveben betyder at funktion virker efter hensigten - se vedhæftet
[Test af specifikke funktioner.docx](https://github.com/DCRGraphsNet/DCROpenCaseManager/files/3385899/Test.af.specifikke.funktioner.docx)

**Comment by DCRGraphs** [comment]
Tak. Forstår ikke hvad der mangler/er forkert. Foreslår vi nummerer ovenstående 1-4.
Ad 1 - virker - ok
Ad 2 - denne er identisk med Ad 3
Ad 3 - flytter ansvaret for indsatsen og alle aktiviteter.
Ad 4 - denne flytter ansvaret for een aktivitet. Virker det ikke? 99+% sikker på denne virker. Det er kun en aktivitet man flytter.

I tvivl om hvad der ikke virker.

**Comment by NicklasHealySyddjurs** [comment]
Test d.d. fra kl. 11.00 - 11.20
[Test af skift ansvarlige 15.07.2019.docx](https://github.com/DCRGraphsNet/DCROpenCaseManager/files/3391624/Test.af.skift.ansvarlige.15.07.2019.docx)

**Comment by DCRGraphs** [comment]
Muddassar har justeret og test er opgraderet. Har ikke haft tid til at teste selv - gør det torsdag hvis du ikke når det før.

**Comment by NicklasHealySyddjurs** [comment]
Test d.d. fra kl. 10.20 - 10.45
Dokumentation vedhæftet

En gentagende fejl er at der ikke bliver ændret ansvarlig i Acadre og at der ikke bliver oprettet et journalark.

En kritisk fejl - der bliver ændret ansvarlig i Acadre men ikke i OCM se pkt. 1.2

Der er også en ændring til journalnotatet
1. title
2. der kommer ikke sagsnummer eller sagsindhold med. (Sagsnummer og sagsindhold skal undtages ved skift på Løbende Journal)

test barn: http://opencasemanagertest.syddjurs.dk/Child?id=727172
test indsats: http://opencasemanagertest.syddjurs.dk/ChildInstance?id=97&ChildId=727172#

[Test af skift ansvarlige 16.07.2019.docx](https://github.com/DCRGraphsNet/DCROpenCaseManager/files/3396152/Test.af.skift.ansvarlige.16.07.2019.docx)

**Comment by DCRGraphs** [comment]
Det med at ændre ansvarlig i Acadre - hvornår sker det ikke? Ved (1) ændre ansvarlig for barn eller (2) ændre ansvarlig for indsats?

De andre ting skulle være løst nu.

**Comment by NicklasHealySyddjurs** [comment]
Det sker altid på nær når man skifter på aktivitet, der skal heller ikke skifte i Acadre.

I testen den 16.07.2019, skriver jeg: Er der skiftet ansvarlig på indsatsagerne på barnet siden?
Ja på nær en, som ikke var amma, den er dog skiftet i Acadre.

Så her har det være en anden ansvarlig på indsatsssagen end amma og derfor ikke skiftet i OCM, men har skiftet den i Acadre.
Så kommer der til at være uligheder mellem OCM og Acadre på sagerne, som vi ikke skal have.

Det eneste sted hvor OCM skal have data på hvem der er ansvarlig er på aktiviteterne ellers er det data fra Acadre.


Venlig hilsen

Nicklas Pape Healy
Sekretariatsmedarbejder
Syddjurs Kommune, Social og Familie sekretariat

Direkte +4587535105
Mobil +4540124834
npth@syddjurs.dk<mailto:npth@syddjurs.dk>
Fra: DCR Solutions <notifications@github.com>
Sendt: 18. juli 2019 12:38
Til: DCRGraphsNet/DCROpenCaseManager <DCROpenCaseManager@noreply.github.com>
Cc: Nicklas Pape Healy <npth@syddjurs.dk>; Comment <comment@noreply.github.com>
Emne: Re: [DCRGraphsNet/DCROpenCaseManager] Hvordan skifter man den ansvarlige for processen? (#327)


Det med at ændre ansvarlig i Acadre - hvornår sker det ikke? Ved (1) ændre ansvarlig for barn eller (2) ændre ansvarlig for indsats?

De andre ting skulle være løst nu.

**Comment by NicklasHealySyddjurs** [comment]
Test d.d. fra kl. 10.40 - 11.16
Dokumentation: [Test af specifikke funktioner 22.07.2019.docx](https://github.com/DCRGraphsNet/DCROpenCaseManager/files/3416565/Test.af.specifikke.funktioner.22.07.2019.docx)

1.  Er på plads, når  journalnotatskabelonen er på plads. Overskriften skal være ”Socialrådgiver skift” og de engelske ord skal oversættes.
2. Her skiftes ansvarlig på alle sagerne. Der skal kun skiftets ansvarlig på indsatssagen.
3. Her skiftes ansvarlig på alle sagerne. Der skal kun skiftets ansvarlig på indsatssagen.
4. Er på plads.

En fælles kritisk fejl er at, hvis der bliver skifter ansvarlig på en udført aktivitet, skiftes ansvarlig også i ”Afsluttede aktiviteter".  "Afsluttede aktiviteter" fungere som en log, og kan under ingen omstændigheder skal kunne ændres på.

**Comment by NicklasHealySyddjurs** [comment]
Test d.d. fra kl. 9.45 - 10.04
Dokumentation: [Test af skift ansvarlig 24.07.2019.docx](https://github.com/DCRGraphsNet/DCROpenCaseManager/files/3426564/Test.af.skift.ansvarlig.24.07.2019.docx)

💯
